recipes/fenics/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+
+if [[ "$(uname)" == "Darwin" ]]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.9
+  export CXXFLAGS="-std=c++11 -stdlib=libc++ $CXXFLAGS"
+fi
+
+for pkg in dijitso ufl instant fiat ffc; do
+    echo "installing ${pkg}-${PKG_VERSION}"
+    git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
+    pushd $pkg
+    pip install --no-deps .
+    popd
+done
+
+pkg=dolfin
+echo "installing ${pkg}-${PKG_VERSION}"
+git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
```

Comment:
The issue here is that this is a single conda package for a multi-repo project, so there is no single source. I tried making the 'main' source the dolfin repo, but since it contains LFS files, conda-build can't clone repos with LFS files right now (https://github.com/conda/conda-build/issues/1462).

I asked @jakirkham about multiple sources, and he said that running git in build.sh was ~the only option for now while we wait for conda-build to support multiple sources (https://github.com/conda/conda-build/issues/1466).